paymail.go

```diff
@@ -157,7 +157,7 @@ func startP2PTransaction(client paymail.ClientInterface,
 // finalizeP2PTransaction will notify the paymail provider about the transaction
 func finalizeP2PTransaction(ctx context.Context, client paymail.ClientInterface, p4 *PaymailP4, transaction *Transaction) (*paymail.P2PTransactionPayload, error) {
 	if transaction.client != nil {
-		transaction.client.Logger().Info(ctx, fmt.Sprintf("finalizeP2PTransaction(): start %s for TxID: %s", p4.Format, transaction.ID))
+		transaction.client.Logger().Info().Msgf("finalizeP2PTransaction(): start %s for TxID: %s", p4.Format, transaction.ID)
```

Comment:
`finalizeP2PTransaction()` can be added to `Str()`.
`transaction.client.Logger().Info().Str('finalizeP2PTransaction()').Msgf("start %s for TxID: %s", p4.Format, transaction.ID)`

Reply:
Removed function name from logs